### DISCUSSION TITLE
Problem at boot with Realtek NIC

### DISCUSSION THREAD

**Discussion by gfreccia** [ltsp/ltsp#671] [General]
Hi All,

I've a Fujitsu laptop with _Realtek SemiconductorRTL8111/8168/8411 PCI Express Gigabit Ethernet Controller_, that very often hangs at boot, stopping during img downloading.
When it boots properly then it works correctly.
Problem on the driver at ipxe?

Thank you!

**Comment by alkisg** [comment]
Does the problem happen if you boot it directly with iPXE?
E.g. you can download USB, CD or HD images from https://boot.ipxe.org/

And another thing to try is this: https://github.com/ltsp/ltsp/discussions/276#discussioncomment-102218
